config/lti_dev_key_sample.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "Canvas Course Manager",
+  "scopes": [],
+  "extensions": [
+    {
+      "platform": "canvas.instructure.com",
+      "settings": {
+        "platform": "canvas.instructure.com",
+        "default": "visible",
```

Comment:
I don't think this is needed

Reply:
Done. https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/40/commits/05a1cde52f98dfe7504581dab3fc6005873d06c3